Makefile

```diff
@@ -1,4 +1,6 @@
 data/whitelisted_ext_idefs.json: 
 	python3 wikidatarefisland/run.py --step ss1 --output "whitelisted_ext_idefs.json"
+data/pipe1.json:
```

Comment:
Let's fall in line with the rest and change this to a filename that describes the result.